libs/unity/library/Tests/Runtime/MediaLineTests.cs

```diff
@@ -175,10 +175,12 @@ public void SetSource()
 
             // Set an invalid source (wrong media kind)
             Assert.Throws<ArgumentException>(() => mediaLine.Source = pc_go.AddComponent<DummyAudioSource>());
+
+            yield return null;
```

Comment:
Why? Need comment, especially as this is the last line of the test.

Reply:
This needs to be run on the Unity app thread, and it needs at least one `yield return` to be a valid coroutine, right? Is there a nicer way to make a valid UnityTest?

Reply:
I guess not. Can you document that this is the reason please? Because I would have not thought about it.